src/main/resources/META-INF/plugin.xml

```diff
@@ -9,7 +9,11 @@
 
   <depends>com.intellij.modules.platform</depends>
 
-  <extensions defaultExtensionNs="com.intellij">
+   <!--    Leave this in here so the IDE gives you feedback on-->
+   <!--    What APIs are available-->
+   <idea-version since-build="201.3803.71"/>
```

Comment:
So I almost introduced a compatibility bug. 

If we keep this `since-build` in the plugin.xml the IDE will show you an error if you try to use something that is not available in the specified compatibility range.

I think it would be easier to maintain if we don't patch the since build in the gradle task (remove the gradle setting). That way only once `since-build` has to be maintained. Thoughts? :thinking:

Reply:
Hmmmmm, you're proposing to remove
https://github.com/zd-zero/waifu-motivator-plugin/blob/809c21d1cddda870864dc129751bd579aa0e14be/build.gradle#L43

and use this instead? Is that, right? I think I need to read more about it for the pros and cons if you'd be glad to guide or share that with me?

Reply:
Yeah [here is the tasks in the plugin](https://github.com/JetBrains/gradle-intellij-plugin#tasks). The patchPluginXml comes in during build time and replaces/adds the version in the plugin xml. 

When the `since-build` is in the xml the IDE can do this:

![image](https://user-images.githubusercontent.com/15972415/87837934-ec94ad80-c85a-11ea-9107-77ed0e6a4945.png)

I just want to avoid having to have to maintain the version in more than one place, because it can be forgotten to be updated in more than one place. If we can figure out how to get the above to work with it being in the gradle settings, great!

End goal is I want the IDE to tell me I have problems before hand.

Reply:
I see, so it'll help us to be more productive in developing with incompatibilities of the SDKs. 

> I just want to avoid having to have to maintain the version in more than one place, because it can be forgotten to be updated in more than one place. If we can figure out how to get the above to work with it being in the gradle settings, great!

For now, we'll go with removing `sinceBuild` from the gradle and go with the `since-build` to the IDE, so we can focus on the feature development instead of finding a way to make it work in Gradle when `since-build` in the xml can do it already.

Reply:
I'll remove it for the time being.

If you want to check out the patched plugin xml, you can always just run `./gradlew patchPluginXml` and the result will be in `build/patchedPluginXmlFiles`